server/koppsApi.js

```diff
@@ -173,6 +174,45 @@ async function getKoppsCourseRoundTerms(courseCode) {
   }
 }
 
+/**
+ * This is temporary method to fetch only round id for UG Rest Api.
+ * Because UG Rest Api is using ladok round id in its group names still.
+ * So once it gets updated then this method will be removed.
+ */
+async function getLadokRoundIds(courseCode, semester, applicationCodes) {
+  const { client } = api.koppsApi
+  const uri = `${config.koppsApi.basePath}course/${encodeURIComponent(courseCode)}/courseroundterms`
+  try {
+    const { body } = await client.getAsync({ uri, useCache: true })
+    const { termsWithCourseRounds } = body
+    const selectedTerm = termsWithCourseRounds.find(t => t.term.toString() === semester.toString())
+    const ladokRoundIds = []
+    if (selectedTerm) {
+      const { rounds = [] } = selectedTerm
+      if (rounds && rounds.length > 0) {
+        for await (const round of rounds) {
+          const { ladokUID, ladokRoundId } = round
+          if (ladokUID && ladokUID !== '') {
```

Comment:
`if (ladokUID)` is enough, because it includes checking if the string is empty, so `&& ladokUID !== ''` can be removed and it should be done in all repos, I have noticed it in several places

Reply:
Remove extra and condition to check empty string.